projects/Allwinner/devices/H6/patches/linux/16-orangepi-lite2.patch

```diff
@@ -0,0 +1,172 @@
+--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi	2019-11-06 20:18:47.685583549 +0800
++++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi	2019-11-06 20:21:30.151635477 +0800
+@@ -55,6 +55,11 @@
+ 	status = "okay";
+ };
+ 
++&gpu {
++	mali-supply = <&reg_dcdcc>;
```

Comment:
If you're mentioning power supply here, you have to also add pmic node in this common dtsi. Before you do that, you have to make sure that all boards which include this dtsi uses same or at least similar pmic design.

Anyway, you should check which boards include this dtsi and download schematic for all of them. Then go trough each node you added and check if it make sense for all affected boards. If it does, then add it to common dtsi. Note that common dtsi can also include partial configuration. For example, if enabling GPU makes sense for all boards, but they have different power supply regulator or some board has fixed one, just enable GPU in common dtsi and add mali-supply property in orangepi lite2 (you don't have to fix boards you don't own or you are not interested in). Another example - if all boards use same pmic, but use slightly different regulator arrangement, define only common PMIC properties in common dtsi and add board specific ones to board dtsi. Again, you don't have to fix other boards dtsi if you don't want to.

Sorry to complicate this so much, but unfortunately your board has split DT which complicates things.

Reply:
Sorry, partial configuration is not good for upstream unless all affected DTs have additional properties. I'll take a look which node makes sense in common DT in a day or two.

Reply:
> Sorry, partial configuration is not good for upstream unless all affected DTs have additional properties. I'll take a look which node makes sense in common DT in a day or two.

Sure..Take your time and let me know how I can fine tune the DTs... I have been learning about DTs for quite somewhile but haven't really did anything useful..I am interested to help you and contribute something back to community as well as learn through this process.. Sorry If I am of too much trouble for you..

I will fix the mali-supply part of the dtsi